content/blog/2022/10/2022-10-20-jenkins-election-announcement.adoc

```diff
@@ -0,0 +1,97 @@
+Therefore, we are trusting participants and not requiring that you provide evidence of contribution as part of your voter registration.
+We reserve the right to ban the specific account from the election process if we identify abuse.
+
+The election has 3 stages:
+
+* Nomination and voter registration
+* Voting
+* Results announcement
+
+== Voters
+
+To be eligble to vote, we need to submit a list of email addresses to the link:https://civs.cs.cornell.edu[Condorcet Internet Voting Service].
+We ask that all community members that are interested in voting, and meet the requirements, join the group link:https://community.jenkins.io/g/election-voter-2022[election-voter-2022] on link:https://community.jenkins.io[community.jenkins.io].
+The group will be open to join during the registration period.
+
+== Nominations
+
+Also during this period, we invite community members to nominate candidates by sending a message to the link:https://community.jenkins.io/g/election-committee[election-committee] group, mentioning the position and the motivation.
+On November 9th, the nomination period will end.
+Nominees will be notified, and asked for confirmation that they are interested in running as a candidate.
+The list of candidates will be announced on November 14th.
+
+*Anyone can nominate candidates.*
+
+For the 2022 election, nominations are open for the following positions:
+
+* link:/project/governance/#governance-board[Board members]
+* link:/project/team-leads/#documentation[Documentation Officer]
+* link:/project/team-leads/#events[Events Officer]
+* link:/project/team-leads/#infrastructure[Infrastructure Officer]
+* link:/project/team-leads/#release[Release Officer]
+* link:/project/team-leads/#security[Security Officer]
+
+Additional information about the roles can be found on the link:/project/team-leads/[team leads] page.
+
+== Election
+
+On the 17th of November, candidates will be announced and everybody that registerd will be notified by email to vote using link:https://civs.cs.cornell.edu[civs.cs.cornell.edu].
+Once candidates have been annouinced, registration will be closed.
+The voting deadline is the 1st of December.
+
+== Results
+
+As soon as the votes have been counted, the results will be published.
+The new term will start on December 3rd, at which point the elected members will transition to their new roles.
+
+== Key Dates
+
+* Oct 20: Voter registration and nominations open
+* Nov 14: Nomination deadline
+* Nov 17: Candidates announced, registration closed, voting period begins
+* Dec 02: Voting period ends
+* Dec 07: Results announced
+
+== Key Information
+
+* Candidates can be nominated by anyone, by sending a private message to the link:https://community.jenkins.io/g/election-committee[election-committee] group.
+* Voters must register for the election by joining the public link:https://community.jenkins.io/g/election-voter-2022[election-voter-2022]group no later than *November 17th*.
```

Comment:
```suggestion
* Voters must register for the election by joining the public link:https://community.jenkins.io/g/election-voter-2022[election-voter-2022] group no later than *November 17th*.
```

Reply:
good catch, I hadn't noticed that one. I updated this accordingly and should now render correctly.